doc/docs/master/deploy-manage/deploy/amazon_web_services/aws-deploy-pachyderm.md

```diff
@@ -149,7 +149,19 @@ If you expect your cluster to be very long
 running or scale to thousands of jobs per commits, you might need to add
 more storage.  However, you can easily increase the size of the persistent
 volume later.
-## 3- Deploy Pachyderm
+
+## 3- Set Up Networking Infrastructure
```

Comment:
@tybritten would definitely love your input here. This is obviously very incomplete, but do the high level events sound right?